go.mod

```diff
@@ -59,32 +59,35 @@ replace (
 replace github.com/harvester/harvester => github.com/harvester/harvester v0.0.2-0.20230209065814-98f56beae339
 
 require (
-	github.com/harvester/harvester v1.1.1
-	github.com/harvester/harvester-network-controller v0.3.1
+	github.com/harvester/harvester v1.1.2-rc8
```

Comment:
Is that okay to use rc here?

Reply:
Yeah, it's good to update to v1.2.0. Thanks.